packages/wrangler/src/versions/deploy.ts

```diff
@@ -96,7 +101,8 @@ export const versionsDeployCommand = createCommand({
 		},
 	},
 	positionalArgs: ["version-specs"],
-	handler: async function versionsDeployHandler(args, { config }) {
+	handler: async function versionsDeployHandler(args) {
+		const config = readConfig(args, { useRedirect: true });
```

Comment:
Could this be added as an option to the command definition helpers?

Reply:
Yes but I worried that this would expand the scope of this PR.
I can do it in a follow up?

Reply:
Actually it is added already! `useConfigRedirectIfAvailable`...

Reply:
Updated this command to use it.